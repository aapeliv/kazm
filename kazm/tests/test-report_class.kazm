class Team_Project {
  int num_of_members;
  double cap;
  string course_name;
  string name;

  Team_Project() {
    me.num_of_members = 5;
    me.cap = 1.0;
    me.course_name = "COMS4115: Programming Languages and Translators";
    me.name = "Aapeli";
  }

  ~Team_Project() {
    println("Goodbye from Team_Project");
  }

  void print_info() {
    print(me.name);
    print(" is in the course: ");
    println(me.course_name);
  }
};

int main() {
  Team_Project tp;
  tp.print_info();
}
