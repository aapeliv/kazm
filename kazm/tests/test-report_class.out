Aapeli is in the course: COMS4115: Programming Languages and Translators
Goodbye from Team_Project
